src/neynar-api/v2/client.ts

```diff
@@ -769,21 +759,21 @@ export class NeynarV2APIClient {
    *
    * For more information, refer to the [Neynar documentation](https://docs.neynar.com/reference/user-channels).
    */
-  public async fetchUserChannels(
-    fid: number,
-    options?: {
-      limit?: number;
-      cursor?: string;
+    public async fetchUserChannels(
```

Comment:
In /v2/client and /neynar-api/neynar-api-client files, we are maintaining a section of code for a specific API methods

Search for `-- Channel --` in code base.

Earlier this method was a part of `-- User --` section, Please move it to the Channel section.